docs/HELM.md

```diff
@@ -79,6 +87,36 @@ $ helm uninstall my-release
 
 The command removes all the Kubernetes components associated with the chart and deletes the release.
 
+## Upgrading from the archived charts repository
+
+Charts for the LogDNA Agent were previously available on the [archived "stable" charts repository][helm-stable].
+If you deployed the agent using those helm charts, you can use `helm upgrade` to update your helm installation.
+
+First, locate the release name used to deploy:
+
+```bash
+helm list -A
+```
+
+Next, use the namespace and release name to run the upgrade command:
+
+```bash
+helm upgrade --set logdna.key=$LOGDNA_INGESTION_KEY --set nameOverride=logdna-agent -n my-namespace my-release logdna/agent
+```
+
+In case the upgrade fails due to invalid `spec.selector`, make sure to use the same label:
```

Comment:
This should be less common after we default to "logdna-agent" as app label: https://logdna.atlassian.net/browse/LOG-8944